edit RN v13.0.2 hotfix (#1478)
mia-platform/documentation: docs/release-notes/v13.0.2.md

---
id: v13.0.2
title: Version 13.0.2 Release Notes
sidebar_label: v13.0.2
image: "img/release-note-link-preview.png"
---

_June 18th, 2024_

## Console

### Bug Fix

This version addressed the following bugs:

* 

## How to update your Console

> For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `vX.X.X`.

Reviewer: ```suggestion
For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v13.3.11`.

```

---
id: v13.0.2
title: Version 13.0.2 Release Notes
sidebar_label: v13.0.2
image: "img/release-note-link-preview.png"
---

_June 18th, 2024_

## Console

### Bug Fix

This version addressed the following bugs:

* fixed bug that caused an error on Flow Manager Configurator's configuration conversion
* fixed a bug causing a malfunction on adding new Dashboards to the Project

> * fixed a regression introduced in `v13.0.0` affecting MongoDB Views endpoint routes: all endpoints associated with MongoDB views now correctly expose the appropriate routes based on whether the views are writable or not

Author: ```suggestion
* fixed a regression introduced in `v13.0.0` affecting MongoDB Views endpoint routes: all endpoints associated with MongoDB views now correctly expose the appropriate routes based on whether the views are writable or not. For more info, visit the [related documentation](/development_suite/api-console/api-design/mongo_views.md#expose-through-endpoints
```

---
id: v13.0.2
title: Version 13.0.2 Release Notes
sidebar_label: v13.0.2
image: "img/release-note-link-preview.png"
---

_June 18th, 2024_

## Console

### Bug Fix

This version addressed the following bugs:

* fixed bug that caused an error on Flow Manager Configurator's configuration conversion
* fixed a bug causing a malfunction on adding new Dashboards to the Project

> * fixed a regression introduced in `v13.0.0` affecting MongoDB Views endpoint routes: all endpoints associated with MongoDB views now correctly expose the appropriate routes based on whether the views are writable or not. For more info, visit the [related documentation](/development_suite/api-console/api-design/mongo_views.md#expose-through-endpoints

Author: ```suggestion
* fixed a regression introduced in `v13.0.0` affecting MongoDB Views endpoint routes: all endpoints associated with MongoDB views now correctly expose the appropriate routes based on whether the views are writable or not. For more info, visit the [related documentation](/development_suite/api-console/api-design/mongo_views.md#expose-through-endpoints)

```
